src/CoreFx.Private.TestUtilities/dir.props

```diff
@@ -4,5 +4,8 @@
   <PropertyGroup>
     <AssemblyVersion>1.0.0.0</AssemblyVersion>
     <AssemblyKey>Test</AssemblyKey>
+    <!-- DO NOT ship this as stable for .NET Core 2.0. -->
+    <BlockStable>true</BlockStable>
+    <IsNETCoreApp>false</IsNETCoreApp>
```

Comment:
Don't worry about explicitly setting IsNetCoreApp=false. The BlockStable=true is the only property I was interested in setting.